webapp/Connector/src/app/view/module/DataAvailabilityModule.js

```diff
@@ -142,7 +146,21 @@ Ext.define('Connector.view.module.DataAvailabilityModule', {
                     {
                         ftype: 'grouping',
                         collapsible: false,
-                        groupHeaderTpl: new Ext.XTemplate('{name:htmlEncode}')
+                        groupHeaderTpl: new Ext.XTemplate('{name:htmlEncode}',
+                                '<tpl if="name === \'MAb Administration Studies\'">',
```

Comment:
@XingY - I added it here since MAb learn page has 'Integrated Data' section (previously called 'Data Availability'), and the grouping based on study_type happens here.  Any idea as to where this instruction could be added if not here?

Reply:
I like how this.data.instructionsForAdministration is passed in instead of hardcoded. Would the following modification to cube.js allow pure data driven tpl? 

hasGrouping: true,
                            groupingSubHeaders: {
                                'MAb Administration Studies' : 'Go to Plot to view or Grid to export.  Additional non-integrated data files may be available for download. See study page.',
                                'MAb Characterization Studies' : 'Go to Plot to view or Grid to export.  Additional non-integrated data files may be available for download. See study page.'
                            },